app/services/geographic/determine_ea_area_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "defra_ruby/area"
+
+module Geographic
+  class DetermineEaAreaService < WasteCarriersEngine::BaseService
```

Comment:
The problem I find with DetermineEaAreaService name is that it doesn't suggest what are we using as an input. And that input might be different: postcode, eating and nothing, national grid reference, etc

How about renaming these services e.g. Geographic::DetermineEaAreaFromEastingAndNorthingService,

or 

Geographic::MapEastingAndNorthingToEaAreaService
Geographic::MapPostcodeToEastingAndNorthingService

Reply:
The "Map..." option works for me.